proposals/0110-distingish-single-tuple-arg.md

```diff
@@ -1,10 +1,10 @@
 # Distinguish between single-tuple and multiple-argument function types
 
 * Proposal: [SE-0110](0110-distingish-single-tuple-arg.md)
-* Authors: Vladimir S., Austin Zheng
-* Review Manager: [Chris Lattner](http://github.com/lattner)
-* Status: **Implemented (Swift 4)**
-* Decision Notes: [Rationale](https://lists.swift.org/pipermail/swift-evolution-announce/2016-July/000215.html)
+* Authors: Vladimir S., [Austin Zheng](https://github.com/austinzheng)
+* Review Manager: [Chris Lattner](https://github.com/lattner), [Doug Gregor](https://github.com/DougGregor)
+* Status: **Implemented (Swift 4 beta)**, **Reverted (Swift 4)**
+* Decision Notes: [Implementation Rationale](https://lists.swift.org/pipermail/swift-evolution-announce/2016-July/000215.html), [Reversion Rationale](https://lists.swift.org/pipermail/swift-evolution-announce/2017-June/000386.html)
```

Comment:
if only one review manager possible, I suggest we keep the active one (Doug Gregor)